app/team.go

```diff
@@ -713,11 +713,11 @@ func (a *App) GetTeamMembers(teamId string, offset int, limit int, restrictions
 }
 
 func (a *App) GetTeamMembersByIds(teamId string, userIds []string, restrictions *model.ViewUsersRestrictions) ([]*model.TeamMember, *model.AppError) {
-	result := <-a.Srv.Store.Team().GetMembersByIds(teamId, userIds, restrictions)
```

Comment:
You can just `return a.Srv.Store.Team().GetMembersByIds(teamId, userIds, restrictions)`

Reply:
Thanks for all these revisions, I forgot several :(